README.md

```diff
@@ -96,6 +96,8 @@ otherwise the polyfilled `fetch` will be installed during the first pass of the
 
 If set to `false`, the polyfilled `fetch` will replace native `fetch` be there or not.
 
+If all your browser targets (defined in `config/targets.js`) support native `fetch`, and `preferNative: true`, the polyfill will not be included in the output build. If, for some reason, you still need the polyfill to be included in the bundle, you can set `alwaysIncludePolyfill: true`.
```

Comment:
```suggestion
 If all your [browser targets](https://guides.emberjs.com/release/configuring-ember/build-targets/) support native `fetch`, and `preferNative: true`, the polyfill will not be included in the output build. If, for some reason, you still need the polyfill to be included in the bundle, you can set `alwaysIncludePolyfill: true`.
```

I think the first paragraph
> If set to `true`, the the fetch polyfill will be skipped if native `fetch` is available, ...

can just be replaced by your added paragraph.

Reply:
Hmm, but that would then not cover the case that `preferNative: true`, but some targets do not have `fetch` support (which, for now, will probably be the case for many apps) - in that case, the setting will still have an effect (--> not use the polyfill if fetch is available).